sdk/src/beta9/abstractions/function.py

```diff
@@ -147,8 +148,12 @@ def __call__(self, *args, **kwargs) -> Any:
         ):
             return
 
-        with terminal.progress("Working..."):
-            return self._call_remote(*args, **kwargs)
+        try:
+            with terminal.progress("Working..."):
+                return self._call_remote(*args, **kwargs)
+        except KeyboardInterrupt:
+            terminal.error("Exiting Shell. Your function will continue running remotely.")
+            os._exit(0)
```

Comment:
`terminal.error` will call `sys.exit(1)`.

Just wondering what the behavior would be if we just did something like `return terminal.error("...")`. Does it exit cleanly and quickly?

Reply:
I removed the exit call. Still exists cleanly. The `terminal.error` call doesn't return anything so I'm just calling it, not returning it

Reply:
Got it. I usually try returning because the linter doesn't know its exiting and will make some incorrect assumptions about other vars and what not. Since this is the end of the function, it should be fine.